changes/19143-mdm-cmd-filters

```diff
@@ -0,0 +1,2 @@
+- `fleetctl get mdm_commands` now returns 20 rows, supports a `--host-identifier` filter, and
+  pagination using `--page` to improve response time
```

Comment:
no `--page`?

Reply:
oh good catch, the scope changed, so i need to pull that out of the release notes